source/inochi2d/core/nodes/composite/package.d

```diff
@@ -254,6 +258,10 @@ protected:
         if (!data["screenTint"].isEmpty) deserialize(this.screenTint, data["screenTint"]);
         if (!data["blend_mode"].isEmpty) data["blend_mode"].deserializeValue(this.blendingMode);
         if (!data["masks"].isEmpty) data["masks"].deserializeValue(this.masks);
+        propagateMeshGroup = false;
```

Comment:
false should be the default state, this is kind of unneccesary

Reply:
Looking through it may be needed, but then at the very least store it at the top of the function.